casemanagementv1/case_management_v1_examples_test.go

```diff
@@ -114,9 +119,15 @@ var _ = Describe(`CaseManagementV1 Examples Tests`, func() {
 
 			createCaseOptions := caseManagementService.NewCreateCaseOptions(
 				"technical",
-				"testString",
-				"testString",
+				"Example technical case",
+				"This is an example case description.",
 			)
+			createCaseOptions.SetSeverity(1)
+			createCaseOptions.SetEu(&casemanagementv1.CasePayloadEu{Supported: core.BoolPtr(true), DataCenter: core.Int64Ptr(123)})
+
+			offeringType, _ := caseManagementService.NewOfferingType(casemanagementv1.OfferingTypeGroupCRNServiceNameConst, "cloud-object-storage")
```

Comment:
I think it would be good to move the construction of the dependent models (offeringType and offeringPayload) to immediately before the construction of the options model.  So the general pattern would be to construct dependent models first, then construct the options model and set various fields on it.